test/functional/web/safari-window-e2e-specs.js

```diff
@@ -99,6 +107,10 @@ describe('safari - windows and frames', function () {
       });
 
       it('should be able to open and close windows', async function () {
+        if (process.env.CI && isIosVersionBelow('18.0')) {
```

Comment:
The current Xcode 16 env can check these behaviors. This will be skipped on Xcode 15 env as pretty much unstable.